### PR TITLE
Fix syntax error in rst list

### PR DESCRIPTION
**Pull request by jeremy-audet**

### CODE REVIEW

File: docs/plugins.rst [hunk 1]
```diff
@@ -14,10 +14,10 @@ about which plugin it should run and Receptor will import and call the requested
 Plugins don't specifically have to be aware that Runner is operating under asyncio, instead
 they are launched using a thread. Each plugin is given three parameters
 
-* The *message* data sent from the Controller
-* Any *configuration* specifically defined for it, which includes some details about the receptor
+*   The *message* data sent from the Controller
+*   Any *configuration* specifically defined for it, which includes some details about the receptor
     node that sent the message
```
Comment (elyezer) [inline docs/plugins.rst]: If you align this with just two spaces in the beginning it should remove the syntax error and avoid having to change previous lines

File: docs/plugins.rst [hunk 2]
```diff
@@ -14,10 +14,10 @@ about which plugin it should run and Receptor will import and call the requested
...
-* A *queue* to place responses into. These responses are delivered back to the Controller
+*   A *queue* to place responses into. These responses are delivered back to the Controller
     immediately when they are placed into the queue
```
Comment (elyezer) [inline docs/plugins.rst]: If you align this with just two spaces in the beginning it should remove the syntax error and avoid having to change previous lines

Comment (jeremy-audet) [reply docs/plugins.rst]: Done!

File: docs/plugins.rst (hunk 1 above)
Comment (jeremy-audet) [reply docs/plugins.rst]: Done!

**Review by elyezer — APPROVED** [review]

**Review by matburt — APPROVED** [review]

**Pull request merged**